### PR TITLE
Sebastian Herrera sort folders and upload the challenges 

### PR DESCRIPTION
**Pull request by elrocetas**
I sorted my folders and upload my challenges

### CODE REVIEW

File: SebastianHerrera/bootcamp-java/src/main/java/hackerrank/day2/exercise1/Bird.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package hackerrank.day2.exercise1;
```
Comment (luisod78) [inline SebastianHerrera/bootcamp-java/src/main/java/hackerrank/day2/exercise1/Bird.java:1]: Very good arranging your classes into packages
Great!

File: SebastianHerrera/bootcamp-java/src/main/java/hackerrank/day2/exercise1/Bird.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+ * @author Sebastian Herrera (bermeosebas@gmail.com)
+ * @created 27/08/21
+ */
+public class Bird extends Animal {
+
+    void fly() {
+        System.out.println("I am flying");
+    }
+    void sing() {
+        System.out.println("I am singing Cielito Lindo");
```
Comment (luisod78) [inline SebastianHerrera/bootcamp-java/src/main/java/hackerrank/day2/exercise1/Bird.java:13]: Canta y no llores

File: SebastianHerrera/bootcamp-java/src/main/java/hackerrank/day2/exercise2/Operations.java [hunk 3]
```diff
@@ -0,0 +1,11 @@
+package hackerrank.day2.exercise2;
+
+/**
+ * @author Sebastian Herrera (bermeosebas@gmail.com)
+ * @created 27/08/21
+ */
+class Operations {
```
Comment (luisod78) [inline SebastianHerrera/bootcamp-java/src/main/java/hackerrank/day2/exercise2/Operations.java:7]: "1. A class named Arithmetic with a method named add that take 2 integers....."
"2. A class named Adder that inherits from....."

Operations???

File: SebastianHerrera/bootcamp-java/src/main/java/hackerrank/day2/exercise3/Exercise3.java [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    public static final String FORMAT_ODER = "%-15s%03d\n";
+
+    /**
+     * Get information from user and formatting
+     */
+    public static void getScanInfo() {
+        Scanner sc = new Scanner(System.in);
+        int[] arrayInteger = new int[3];
+        String[] arrayString = new String[3];
+        for (int i = 0; i < 3; i++) {
```
Comment (luisod78) [inline SebastianHerrera/bootcamp-java/src/main/java/hackerrank/day2/exercise3/Exercise3.java:21]: Even when all programing books include examples of for loops with var names like i, x, z, and so on, please avoid these.

HINT: (1 y 2) https://javarevisited.blogspot.com/2014/10/10-java-best-practices-to-name-variables-methods-classes-packages.html

**Review by luisod78 — CHANGES_REQUESTED** [review]

**Review by luisalanmz — APPROVED** [review]
Ok

**Review by edbermeo — APPROVED** [review]
LGTM

**Pull request merged**